config/kernel-get-disk-and-module.m4

```diff
@@ -0,0 +1,19 @@
+dnl #
+dnl # 4.16 API change
```

Comment:
nit: i think this change landed in linux-block/for-4.17

Reply:
I was looking at https://github.com/torvalds/linux and they are currently in `4.16-rc` releases. Am I looking at the wrong place for version numbers?

Reply:
https://stackoverflow.com/questions/30268332/why-does-the-linux-kernel-repository-have-only-one-branch

Reply:
@dinatale2 i found the change here http://git.kernel.dk/cgit/linux-block/log/?h=for-4.17/block (http://git.kernel.dk/cgit/linux-block/commit/?h=for-4.17/block&id=3079c22ea815775837a4f389ce2f7e1e7b202e09) and thought this was going to land in 4.17. It's just a nit though, the functional aspect is the important one and LGTM.